backend/src/main/java/ai/verta/modeldb/experiment/ExperimentServiceImpl.java

```diff
@@ -162,7 +164,8 @@ public void createExperiment(
           ModelDBServiceActions.CREATE,
           Collections.singletonMap(experiment.getId(), entityResource.getWorkspaceId()),
           ModelDBUtils.getStringFromProtoObject(request),
-          ModelDBUtils.getStringFromProtoObject(response));
+          ModelDBUtils.getStringFromProtoObject(response),
+          entityResource.getWorkspaceId());
```

Comment:
Workspace service id? Please check everywhere as I think some other cases also use this old id instead of the new one.

Reply:
No, if we are getting from entity then it should be `workspace_service_id`  like `project.getWorkspaceServiceId()` already i have used. but when we are getting it from `GetResourcesResponseItem` then there only single field which is `workspace_id` like `entityResource.getWorkspaceId()`

Reply:
Oh, got it.